scripts/migrateHistoricBids.ts

```diff
@@ -68,13 +68,13 @@ async function getAllFromCollection<T>(
   return result;
 }
 
-async function writeMessagesToOutputFile<T>(messages: TypedMessage<T>[][]) {
+async function writeMessagesToOutputFile<T>(messages: TypedMessage<BidPlacedV1Data | BidCancelledV1Data>[][]) {
   var flat = messages.flat();
```

Comment:
Don't use `var`, use `let` or `const`

Reply:
Also `.flat()` requires `es2019` or better in `tsconfig.json` please update the `lib` field in that file to be `es2019` instead of `es2018`. I'm surprised tsc didn't catch this.